packages-ts/gauntlet-terra-contracts/src/commands/wallet/send.ts

```diff
@@ -0,0 +1,62 @@
+import { BN, prompt } from '@chainlink/gauntlet-core/dist/utils'
+import { AccAddress, MsgSend } from '@terra-money/terra.js'
+import { CATEGORIES, ULUNA_DECIMALS } from '../../lib/constants'
+import { TerraCommand, TransactionResponse } from '@chainlink/gauntlet-terra'
+import { Result } from '@chainlink/gauntlet-core'
+
+type CommandInput = {
+  destination: string
+  // Units in LUNA
+  amount: string
+}
+
+export default class TransferLuna extends TerraCommand {
+  static description = 'Transfer Luna'
+  static examples = [`yarn gauntlet wallet:transfer --network=bombay-testnet`]
+
+  static id = 'wallet:transfer'
+  static category = CATEGORIES.WALLET
+
+  input: CommandInput
+
+  constructor(flags, args: string[]) {
+    super(flags, args)
+  }
+
+  buildCommand = async (flags, args): Promise<TerraCommand> => {
+    this.input = this.makeInput(flags, args)
+    return this
+  }
```

Comment:
Now you have two construction steps. Why not just do all of this in the constructor?
Designing it like this it's easy to leave the object in a half-constructed state.

Reply:
We need a way to construct the object with support for promises basically. Currently `gauntlet-core` initializes the command using `new Command`, but we should move into a `static create` (or `buildCommand`) style, and be the default way to construct the command.